### PR TITLE
generateRandom function

### PR DESCRIPTION
**Pull request by aaronli03**
## What
- modified generateRandomString to generateRandom
- generateRandom() can be used to construct a state or code verifier
- Changed "Format with Prettier" title to "Check with Prettier"

## Why
- this single function compacts two fuctions into one
- new arg will determine which character set to use
- changed name to correctly represent the intention of command

## Impact
- Will customer experience be significantly altered? No
- Does the code change cover more than one task? No

## Testing
- tested locally by running `npm run dev`

### CODE REVIEW

**Comment by aaronli03** [comment]
Next pr will address the prettier format warn.

File: src/api/auth.ts [hunk 1]
```diff
@@ -1,13 +1,15 @@
 import * as crypto from 'crypto';
 
-export function generateRandomString(length: number) {
-	const characterSet = 'ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789';
-	let randomString = '';
+export const stringSet = 'ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789';
+export const codeVerifierSet = 'ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789-._~';
```
Comment (byjackli) [inline src/api/auth.ts]: do uppercase `STRING_SET` and `CODE_VERIFIER_SET`

Comment (byjackli) [reply src/api/auth.ts]: make sure to update your tests to check against both different sets

**Review by byjackli — CHANGES_REQUESTED** [review]
squash all your 3 commits into 1. each PR should only have ONE commit

**Review by byjackli — APPROVED** [review]
In the future, instead of using `* as` can you try just importing the specifics you need?

**Pull request merged**